XenModel/Messages.resx

```diff
@@ -13048,6 +13048,12 @@ Please check your settings and try again.</value>
   <data name="UPDATES_WIZARD_WLB_ON_WARNING" xml:space="preserve">
     <value>{0}: Check skipped because WLB is enabled on pool {1}.</value>
   </data>
+  <data name="UPDATE_CLIENT_CONFIRMATION_MESSAGE" xml:space="preserve">
+    <value>In order to update Citrix Hypervisor Center will be closed. Please make sure you've completed and saved your work.</value>
+  </data>
+  <data name="UPDATE_CLIENT_CONFIRMATION_MESSAGE_TITLE" xml:space="preserve">
+    <value>Update Citrix Hypervisor Center</value>
+  </data>

```

Comment:
We should use `BrandingManager` to fetch the name of the product, like we do in other parts of the app. So here it should be `in order to update {0}` and `Update {0}`

Reply:
Did not know that was a thing, will do